backend/routes/api/export.js

```diff
@@ -416,6 +455,26 @@ router.get(
         // Get the Feature for this DateRange, if there is one
         const feature = getFeatureForDateRange(dateRange);
 
+        // Get the GateDetail display values
+        const {
+          isGateType,
+          hasGate,
+          gateStartTime,
+          gateEndTime,
+          sameEveryYear: gateSameEveryYear,
+        } = getGateDisplayValues(dateRange, season.gateDetail, annualData);
```

Comment:
```suggestion
        } = getGateDisplayValues(dateRange, gateDetail, annualData);
```
We already have `gateDetail` from the section above, so we could simplify this, but otherwise this looks perfect 👍